.gitignore

```diff
@@ -48,3 +48,9 @@ test/nodal-dg
 # Emacs backups
 .\#*
 \#*
+
+
+# disk caches
+xdg-cache/
+pocl-cache/
+cuda-cache/
```

Comment:
I'm forgetting if this applies to subdirs of the repo too as written, or if only the top level dir? (I sometimes run from `examples/`, so the former would be helpful.)

Reply:
It should apply to subdirs.